inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -256,6 +261,43 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
+            std::vector<std::string> value;
+            pugi::xml_node dn = node.child("data");
+            auto type = XMLParseUtils::GetStrAttr(node, "type");
+
+            if (dn.empty())
+                THROW_IE_EXCEPTION << "No attrtibutes defined for " << type << " op!";
+
+            if (getParameters<std::string>(dn, name, value)) {
```

Comment:
Why do you read value as vector of strings? I thought that it should be one buffer. Moreover in next lines you work only with first element from this vector

Reply:
Corrected.